### PR TITLE
Updates CONTRIBUTING with agreed workflow and adds CODE_OF_CONDUCT

### LINKED ISSUE #96: Update CONTRIBUTING.adoc and add CODE_OF_CONDUCT.adoc
`CONTRIBUTING.adoc` should be expanded to include other conventions agreed upon in the "Spring on GCP Workflow Proposal" doc.

`CODE_OF_CONDUCT.adoc` sould be added with content similar to [code of conduct](https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/CODE_OF_CONDUCT.md).

### PR DESCRIPTION
**Pull request by joaoandremartins**
Fixes #96

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -26,3 +27,36 @@ Settings`, it will set this policy across all of your IDEA projects.
 * IDEA's "Optimize imports on the fly" option turned on interferes with the Eclipse code formatter
 import optimization. Consider disabling the option if optimization does not yield the expected
 results.
+
+= Contribution workflow
```
Comment (meltsufin) [inline CONTRIBUTING.adoc]: I think we need to make it clear that this workflow applies to project committers.

Community contributions should follow a workflow like this:

# Contributing A Patch

* Submit an issue describing your proposed change to the repo in question.
* The repo owners will respond to your issue promptly.
* If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement [link].
* Fork the repo, develop and test your code changes.
* Ensure that your code adheres to the existing style in the sample to which you are contributing. Refer to the Spring Style Guide [link] for the recommended coding standards for this organization.
* Ensure that your code has an appropriate set of unit tests which all pass.
* Submit a pull request.

Comment (artembilan) [reply CONTRIBUTING.adoc]: +1. We don't need to tell target contributor how to press "squash & merge".

Comment (joaoandremartins) [reply CONTRIBUTING.adoc]: If the workflow in this PR only applies to project committers, I wonder if it's worth to have it in here at all? After all, projects committers should have access to the Google Doc the workflow was based in.

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -28,35 +28,22 @@ Settings`, it will set this policy across all of your IDEA projects.
...
-8. Protect master branch from direct commits.
-
-9. Use Github's
-https://help.github.com/articles/configuring-commit-squashing-for-pull-requests/[squash & merge
-feature] when submitting PRs and enforce it through Github
-
-10. When merging the PR, use the PR Title + Description as the commit message. See sample in the Appendix below.
+6. Ensure your code has an appropriate set of unit tests which all pass
 
+7. Submit a pull request. After it is approved by a project committer, you can Squash & Merge the
```
Comment (meltsufin) [inline CONTRIBUTING.adoc]: They won't have the permissions to merge.

**Pull request merged**